frontend/hub/approvals/hooks/useApprovalsColumns.tsx

```diff
@@ -41,14 +40,18 @@ export function useApprovalsColumns(_options?: { disableSort?: boolean; disableL
             return (
               <TextCell
                 icon={<ExclamationTriangleIcon />}
-                text={t('Needs review')}
+                text={
+                  approval.is_signed === false && can_upload_signatures && require_upload_signatures
+                    ? t`Needs signature and review`
+                    : t`Needs review`
+                }
                 color={PFColorE.Warning}
               />
             );
           }
 
-          if (approval.repository?.pulp_labels?.pipeline == 'approved') {
-            if (approval.is_signed && display_signatures) {
```

Comment:
probably not meant to remove this?

if display_signmatures is false, this needs to show only Approved